lib/sandbox.sh

```diff
@@ -27,7 +27,7 @@ bundle exec rails new sandbox --database="$RAILSDB" \
   --skip-rc \
   --skip-spring \
   --skip-test \
-  --skip-yarn
+  --skip_javascript
```

Comment:
Shouldnt this be?

```suggestion
  --skip-javascript
```

Reply:
Yes but I swear that I tried and it worked the same! 😅 Thanks!